### DISCUSSION TITLE
Update the pharmaverse.org landing page {crowdsource initiative}

### DISCUSSION THREAD

**Discussion by MichaelRimler** [pharmaverse/pharmaverse#192] [Ideas]
Imagine you were new to pharmaverse and you navigated your browser to pharmaverse.org. What would you expect to see? What would you expect to learn in the first few seconds at the site?

Currently, the website displays a subset of packages that are in the pharmaverse. Is this what we want? Might it induce a bias to the interpretation of what the pharmaverse is - or even WHY the pharmaverse is? Could we do better? 

**Use this issue to post your thoughts** (including mockups/prototypes of images) for the landing page - even if you think it should stay the same. The website product owner (@rossfarrugia) will review all submissions and determine how to make a decision - ultimately implementing.

In your consideration - think whether the website should be conveying WHAT the pharmaverse is - or the HOW or the WHY? And what image would accomplish that - your vision of what we want to communicate in the first 15 seconds of engagement when someone clicks to pharmaverse.org.

**Comment by MichaelRimler** [comment]
One suggestion from @NNaikp in Issue https://github.com/pharmaverse/pharmaverse/issues/188

**Reply by epijim** [reply]
I can't remember if I was told this, or they asked me directly - but someone mentioned to someone 😆 (and it got back to me) that they wanted to know if the 6 hex stickers on the front page were the official packages (e.g. more important than the others). 

So I think at a minimum we either add all the packages to the wall, remove it, or do something to make it more fair (e.g. 'this months featured packages', and have a script to rotate them). I much prefer just making it smaller hexes and adding them all though.

**Reply by MichaelRimler** [reply]
I still think this leads with the WHAT and not the WHY. That type of graphic has its value, but maybe not as the very first thing people see. But, this is my opinion - and I haven't come up with a good alternative yet (admittedly).

**Reply by ari-stotle-max** [reply]
After reading Michael's opening and your comments [below](https://github.com/pharmaverse/pharmaverse/discussions/192#discussioncomment-4902698) I had this idea about just making the logo into a div flip on hover as the logo itself is made from hexes. So it would be something like:

landing: pharmaverse logo
hover 1: Flip to show hexes from pharmaverse (hexes shown could rotate)
hover 2: Show small hexes with company logos comitted
hover 3: Show many small avatar hexes with contributors
hover 4: Show one big hex with something like: Your contribution?
return to landing.

Then maybe with some stats (3?) on each side (here in no particular order):
- no of contributors
- no of packages
- no of commits
- ...

I don't know if it's possible to imagine this, but if it sounds appealing I could try making a mock.

**Reply by rossfarrugia** [reply]
Might well be the only way to achieve this if we end up trying to go with multiple of these ideas, as we potentially have so many great things to share on the homepage. I'm always conscious of overwhelming people too though and scaring them off from diving deeper into the site if too much gets thrown at them from the first page.

**Comment by epijim** [comment]
I think a common confusion is people assume the pharmaverse is like the tidyverse. e.g. super opinionated and top down architecture, and there is one primary way of doing things and an overall architect thinking end to end - with methods actively deprecated (e.g. DBI, reshape2) rather than having duplicated functionality.

I think what we have is very different, as there is no one entity saying 'this is the vision of how things flow x to y', we have many different paths. It doesn't really touch on that on the home page at all.

**Reply by rossfarrugia** [reply]
agree - this comes up time and again! someone asked the other day _"if i adopt the pharmaverse does that mean i can't use any other R packages outside of this?"_. there are so many things to unpack in that question given nobody would ever "adopt the pharmaverse" in totality - companies need to make their own choice of what they need, most of the packages we recommend are dependent on so many other R packages, and pharmaverse is such a targeted niche of e2e clinical reporting. i also hear the confusion with tidyverse often too!

**Reply by epijim** [reply]
Would adding just a paragraph of text fix this? e.g. extend the initial text a little, or make the first Q&A about this? I guess it's in the charter, but I feel many people might not get that far.

**Reply by MichaelRimler** [reply]
It might - but I'd steer towards thinking of the landing page as a blank slate. Instead of micro fixes here and there - what is the message we want to land on this page? Part of the PURPOSE of the pharmaverse is to consolidate the set of packages one _could_ use for reporting - but it is by no means a rule of law. I agree this is not clear in the landing page and perhaps should be in some way.

**Reply by ari-stotle-max** [reply]
I think first step would be to remove the tidyverse-esque opening sentence on the page. Maybe just keep the last part: "...a community of passionate individuals and organisations committed to co-creating efficiency in our mission to improve health."

**Reply by rossfarrugia** [reply]
Yep, think we could re-look at that statement. I still think the curated stack of packages is an important piece of what we offer that should be called out, but people could then rightfully ask why are we not just a CRAN task view (we heard this challenge recently!). Really i think its the spirit of community and influencing co-creation which is the heart of the pharmaverse, and we do share some recommended packages but anyone can choose whichever they like, and we're never realistically going to make every pharma package "share an underlying design philosophy, grammar, and data structures" - to quote tidyverse.

**Reply by mstackhouse** [reply]
I want to support @MichaelRimler's point that I think front and center what we should put forward is _why_ the pharmaverse is a thing - we're aiming to provide solutions and pathways to supporting the clinical reporting pipeline in R - in a collaborative way - where organizations can come together to build more robust solutions to shared problems. 

To @NNaikp's point, I completely agree that we need to drop the tidyverse-esque statement, because it's misleading. We're not as opinionated as the tidyverse, and people come expecting to see a "Golden path". We'll realistically be able to offer path_ways_ where we have multiple solutions and ways to navigate the pipeline. But again - that draws back to the problem we're trying to solve and what we're trying to offer, and why we started trying to do this in the first place.  

From my perspective, it's essential that we communicate this element. Because I think this is where we're really failing a lot of communication right now.

**Comment by epijim** [comment]
Declaring conflict as I added this 😆, but I really like the contributors - where we programatically capture the people that have contributed to a pharmaverse package and then list them. Helps to flag how the pharmaverse is code, and that code comes from currently 122 people spread across many companies. I wonder if we could have more, and push the 'to be part of the pharmaverse as an individual, all you need to do is contribute to an issue' (we also curate all those open issues here: http://openpharma.pharmaverse.org/?nav=openissues)

**Reply by mstackhouse** [reply]
I can support this because it helps steer us away from unintentional bias and over-focus on packages. 

Putting my creative hat on - what if we had a graphic of GitHub profile pictures from the different contributors - think like the apple watch app menu where it zooms out but when you move around it magnifies those pictures.

**Reply by ari-stotle-max** [reply]
> Putting my creative hat on - what if we had a graphic of GitHub profile pictures from the different contributors - think like the apple watch app menu where it zooms out but when you move around it magnifies those pictures.

This sent me spiralling into a dark hole trying to find the CSS/JS that could do this 😅
I did find something else pretty cool which I experimented with [here](https://codepen.io/NNaikp/pen/XWPNVXV).
Without being to artsy it's supposed to represent that pharmaverse is made up of individuals connecting (and the simple node/line combo makes me think of the commit history).

Just an experiment 🕺

**Reply by rossfarrugia** [reply]
I like it! OS as art & speaks to the connection of contributors across packages. shows pharmaverse as more of a joined up familial effort, rather than simply a list of completely independent packages ❤️

**Comment by rossfarrugia** [comment]
@NNaikp had an idea here around sharing LinkedIn posts with #pharmaverse tag too to highlight the community side: https://github.com/pharmaverse/pharmaverse/issues/173#issuecomment-1424064857

**Comment by rossfarrugia** [comment]
See latest Issues for all the suggested updates to the homepage as a result of this dicussion - thanks to those that shared their thoughts! We'll work on implementing the updates staggered over the coming months.
